test/flycheck-test.el

```diff
@@ -4489,6 +4489,30 @@ Why not:
      '(4 3 error "Expected indentation of 2 characters"
          :checker javascript-jscs))))
 
+(flycheck-ert-def-checker-test javascript-standard javascript nil
+  :tags '(checkstyle-xml)
+  (let ((flycheck-checker 'javascript-standard))
+    (flycheck-ert-should-syntax-check
+     "checkers/javascript-standard.js" '(js-mode js2-mode js3-mode))))
```

Comment:
I think this test case is redundant.  We don't need to test that a checker did nothing.  It's just important to know that it works correctly (e.g. runs, and has its output parsed) if there are any errors in a file (since a silent failure is pretty fatal if you rely on Flycheck reporting correct warnings).